tests/test_dataframe.py

```diff
@@ -481,7 +482,7 @@ def tests_merge_left_compound_key(self):
             self.assertEqual(expected, ddf['l_vals'].data[:])
             self.assertEqual(expected, ddf['r_vals'].data[:])
             self.assertEqual(ddf['l_id_1'].data[:].tolist(), ddf['r_id_1'].data[:].tolist())
-            self.assertEqual(ddf['r_id_2'].data[:].tolist(), ddf['r_id_2'].data[:].tolist())
+            self.assertEqual(ddf['l_id_2'].data[:].tolist(), ddf['r_id_2'].data[:].tolist())
```

Comment:
This is still wrong. It should be 'r_id_1' on the left

Reply:
yeah, I think I fixed in the groupby api, just not pushed to master yet

Reply:
It seems to me that we should use 'l_id_2', instead of 'r_id_1'?